### PR TITLE
powerprofilesctl: add page

### PR DESCRIPTION
**Pull request by gotlougit**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/powerprofilesctl.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# powerprofilesctl
+
+> Makes power profiles handling available over D-Bus.
```
Comment (navarroaxel) [inline pages/linux/powerprofilesctl.md:3]:
```suggestion
> Make power profiles handling available over D-Bus.
```

File: pages/linux/powerprofilesctl.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://gitlab.freedesktop.org/hadess/power-profiles-daemon/>.
+
+- List available power profiles:
+
+`powerprofilesctl list`
+
+- Set a particular power profile:
```
Comment (navarroaxel) [inline pages/linux/powerprofilesctl.md:10]:
```suggestion
- Set a specific power profile:
```

File: pages/linux/powerprofilesctl.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`powerprofilesctl set {{arg_value}}`
```
Comment (navarroaxel) [inline pages/linux/powerprofilesctl.md:12]: We use this kind of placeholders:
```suggestion
`powerprofilesctl set {{profile_name}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @gotlougit

**Pull request merged**